### ISSUE TITLE
Login with dynamic passcode

### ISSUE THREAD

**Issue by pwwang** [SchoofsKelvin/vscode-sshfs#19]
It is possible to add an option `tryKeyboard`, just like [`ssh2`](https://github.com/mscdex/ssh2) does if the the server uses dynamic passcode for authentication?
```
login as: xxx
Using keyboard-interactive authentication.
Enter PASSCODE:
```
The passcode is usually a RSA token which changes every minute.

**Comment by SchoofsKelvin** [comment]
That has been something on my list of features to add, and will be added soon.

**Comment by SchoofsKelvin** [comment]
I've implemented the keyboard-interactive/tryKeyboard (0b7e9da), which should allow the server to prompt you for your passcode. This feature is published in version 1.9.0 of the extension.

If you have any troubles related to this, feel free to comment here, but I'll close the issue for now.
